### DISCUSSION TITLE
The ID:make function does not return the right UUID

### DISCUSSION THREAD

**Discussion by Alberina** [laravel/nova-issues#5404] [Q&A] [answered]
Hey all,
An issue is happening only on one of the resources of our app. All the tables are with UUID, not normal integer ID. And it used to work for all, but now it gets only the first character of UUID.

**Comment by crynobone** [comment, marked as answer]
Everything seems to work as expected:


https://user-images.githubusercontent.com/172966/225171132-88fc3069-45a8-4cdd-a9ea-6ad59c346ff6.mp4
